### PR TITLE
Sword jabbing with right-click

### PR DESCRIPTION
**Pull request by NathanHinthorne**

### CODE REVIEW

File: js/scripts/util.js [hunk 1]
```diff
@@ -176,12 +176,21 @@ const checkBlockCollisions = (entity) => {
 /** Contains all functions called as event handlers. */
 const EVENT_HANDLERS = {
     gameplayMouseDown: (e) => {
-        GAME.user.firing = false;
-        GAME.user.aiming = true;
+        // check if mouse button is left click
```
Comment (tclaar) [inline js/scripts/util.js:179]: It's a little confusing to me that the comment says you're checking for a left click, but the first condition is checking for right click. Not a big deal at all though.

**Review by tclaar — APPROVED** [review]
Looks good! We'll still need to make a couple changes to the Sword class to make it work better with right clicking. Thanks for getting this done!

**Review by b1gd3vd0g — APPROVED** [review]
Very nice!

File: js/scripts/util.js [hunk 2]
```diff
@@ -176,12 +176,21 @@ const checkBlockCollisions = (entity) => {
...
+        if (e.button === 2) {
```
Comment (calebkrauter) [inline js/scripts/util.js:180]: Could we update some of the naming here? e.button === 2 is confusing. Perhaps e.click === RIGHT_CLICK or something like that?

**Review by calebkrauter — APPROVED** [review]
Nice work.

**Pull request merged**